whoc/design_tools/wake_steering_design.py

```diff
@@ -0,0 +1,630 @@
+import numpy as np
+import pandas as pd
+from floris import FlorisModel, UncertainFlorisModel, WindRose, WindTIRose
+from floris.optimization.yaw_optimization.yaw_optimizer_sr import YawOptimizationSR
+from scipy.interpolate import interp1d, RegularGridInterpolator
+
+
+def build_simple_wake_steering_lookup_table(
+    fmodel: FlorisModel,
+    wd_resolution: float = 5,
+    wd_min: float = 0,
+    wd_max: float = 360,
+    ws_resolution: float = 1,
+    ws_min: float = 8,
+    ws_max: float = 8,
+    ti_resolution: float = 0.02,
+    ti_min: float = 0.06,
+    ti_max: float = 0.06,
+    minimum_yaw_angle: float = 0.0,
+    maximum_yaw_angle: float = 25.0,
+) -> pd.DataFrame:
+    """
+    Build a simple wake steering lookup table for a given FlorisModel using the Serial Refine
+    method.
+
+    This is mainly intended for demonstration purposes. Users that want more control over the
+    optimization process should use FLORIS's YawOptimizationSR procedure directly to produce
+    the output df_opt.
+
+    Args:
+        fmodel (FlorisModel): An instantiated FlorisModel object.
+        wd_resolution (float, optional): The resolution of the wind direction in degrees.
+            Defaults to 5.
+        wd_min (float, optional): The minimum wind direction in degrees. Defaults to 0.
+        wd_max (float, optional): The maximum wind direction in degrees. Defaults to 360.
+        ws_resolution (float, optional): The resolution of the wind speed in m/s.
+            Defaults to 1.
+        ws_min (float, optional): The minimum wind speed in m/s. Defaults to 8.
+        ws_max (float, optional): The maximum wind speed in m/s. Defaults to 8.
```

Comment:
I was looking into the code and note that use add .001 to ws_max, so it could be worth noting that the arrays will be defined with ws_max inclusive, just to make the behavior clear.  So in the default 
```
>>> np.arange(8,8.001,1)
array([8.])
```